tests/containers/podman_docker_interop.pm

```diff
@@ -0,0 +1,129 @@
+# SUSE's openQA tests
+#
+# Copyright 2024 SUSE LLC
+# SPDX-License-Identifier: FSFAP
+
+# Summary: Validate docker and podman network interoperability
+# - validate if podman and docker networking works together
+# - check if firewalld doesn't break either
+# Maintainer: QE-C team <qa-c@suse.de>
+
+#use Mojo::Base qw(consoletest);
+use Mojo::Base qw(containers::basetest);
+use testapi;
+use serial_terminal 'select_serial_terminal';
+use utils;
+use version_utils qw(check_os_release get_os_release is_sle is_sle_micro is_transactional);
+use transactional qw(check_reboot_changes);
+#use containers::common;
+
+my $test_image = "registry.opensuse.org/opensuse/nginx";
+my $test_egress_cmd = "curl https://opensuse.org/ -vso/dev/null";
+
+sub run {
+    my ($self, $args) = @_;
+
+    select_serial_terminal;
+
+    # Initialize podman
+    $self->containers_factory('podman');
+
+    # Required, otherwise any subsequent outgoing podman connections fail
+    # https://progress.opensuse.org/issues/173362
+    assert_script_run("podman run --rm $test_image $test_egress_cmd");
+
+    # Install firewalld in case it's not installed
+    if (script_run('which firewall-cmd') != 0) {
+        if (is_transactional) {
+            trup_call('pkg install firewalld');
+            check_reboot_changes;
+        } else {
+            zypper_call('in firewalld');
+        }
+    }
```

Comment:
Is this really needed?  I'm 99% sure firewalld is installed by default on all products.

Anyway, we have the `install_packages` routine to deal with installation on both transactional & traditional systems:
https://github.com/os-autoinst/os-autoinst-distri-opensuse/blob/master/lib/containers/common.pm#L272